### DISCUSSION TITLE
Breakpoints to work consistently in VS code.

### DISCUSSION THREAD

**Discussion by pjain11** [microsoft/vscode-react-native#2119] [Q&A] [answered]
Hello,
I'm trying to debug React native code in VS code. I installed this extension and followed all other installation steps from the readme. The breakpoints are not consistently binding. I'm on react native 0.73. 

1. If I start debugger with this tool  (by starting debug from VS code), the app loads fine in simulator but breakpoints binding disappears.   
2. If I start the ios app on simulator by command line and then if start debugger in VS code with react native tools, the debugger is attaching to breakpoints a few times (25% of times). and it works for few code changes and then again just disconnects. 

Do you guys know any consistent way to attach debugger?

**Comment by EzioLi01** [comment, marked as answer]
Hey, actually breakpoint is not handled by this extension. Most of issues for breakpoint is caused by `react-native metro` and vscode built-in extension `vscode-js-debug`. Maybe you can create issue in meta team repo for the help firstly.
